src/main/kotlin/com/bridgecrew/ui/CodeDiffPanel.kt

```diff
@@ -24,7 +24,13 @@ class CodeDiffPanel(val result: BaseCheckovResult, private val isShowBadCode: Bo
                 .inlineDiffByWord(true)
                 .build()
 
-        val rows = generator.generateDiffRows(buildVulnerableLines(), buildFixLines())
+        var oldCode = buildVulnerableLines()
+        var newCode = buildFixLines()
+        if(!isShowBadCode){
```

Comment:
I would either change the name of the boolean to `isErrorBubble` or add a boolean for this case because they are not exactly the same

Reply:
Also you can write it something like `oldCode = if(isErrorBubble) buildFix() else buildLines()